test/Tools/docGeneratorTestFiles/sampledictionarycontent/Dynamo_Nodes_Documentation.json

```diff
@@ -2992,6 +2992,12 @@
     "dynFile": ["Output"],
     "folderPath": "Input/Basic/Action",
     "inDepth": "Output will create an output port for a custom node. The syntax for an output node is simply its name, along with an optional custom comment. In the example below, an output named Percentage is created with a custom comment. It is important to note that this node is only available while creating a custom node."
+}, {
+    "Name": "Custom Dropdown Menu",
```

Comment:
so, this shouldn't be necessary, it's been a long time since I wrote these tests though so maybe I am wrong - BUT it seems a bit backwards that you need to do this as we're hopefully getting rid of this wacky json and instead using the generated .md files and the layoutspec to construct the dictionary one day, no need to change this now though if you found it was required.

Reply:
Yeah, `MarkdownGeneratorCommandTests.DictionaryContentIsFoundCorrectlyForCoreNodes` fails if I don't have this.